io/zenoh-transport/src/unicast/establishment/accept.rs

```diff
@@ -687,7 +682,7 @@ pub(crate) async fn accept_link(link: LinkUnicast, manager: &TransportManager) -
     let iack_out = {
         let mut state = State {
             transport: StateTransport {
-                batch_size: manager.config.batch_size.min(batch_size::UNICAST).min(mtu),
+                batch_size: manager.config.batch_size.min(mtu),
```

Comment:
Don't we risk to forget to reintroduce this `min` if we change `batch_size::UNICAST` ?